tests/utils/test_app_proxy.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+from urllib import parse
+
+import pytest
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+
+from spylib.utils import calculate_message_hmac
+from spylib.utils.app_proxy import CheckAppProxy
+
+API_KEY = "API_KEY"
+API_SECRET = "API_SECRET"
+now = datetime.now()
+
+SHOP = 'test.myshopify.com'
+PREFIX = '/apps/api'
+TIMESTAMP = "1317327555"
+
+
+@pytest.fixture(scope='function', autouse=False)
+def url():
+    params = f'shop={SHOP}&path_prefix={PREFIX}&timestamp={TIMESTAMP}'
+
+    signature = calculate_message_hmac(API_SECRET, params)
```

Comment:
You shouldn't use our code to make the fixture that will test our code ... you are just checking consistency here, not accuracy!
Either copy paste values straight from Shopify or calculate them yourself "manually" with a different code once and hardcode here.
The first option is best but can be annoying to get.